AirportTicketBookingSystem/Domain/Utilities/FakeFileHandler.cs

```diff
@@ -3,14 +3,16 @@
 public class FakeFileHandler : IFileHandler
 {
     private readonly Dictionary<string, List<string>> _files = new();
+    private const string DEFAULT_PATH = "test-file";
 
-    public string[] ReadAllLines(string path) => _files.ContainsKey(path) ? _files[path].ToArray() : new string[0];
+    public string[] ReadAllLines(string path) =>
+        _files.ContainsKey(DEFAULT_PATH) ? _files[DEFAULT_PATH].ToArray() : new string[0];
 
     public void WriteAllLines(string path, IEnumerable<string> lines)
```

Comment:
Cool idea to test against a fake file! Lets also make sure we delete the file/db after the tests are done running. This should be done for both repos (booking and flights).